src/internal.c

```diff
@@ -11196,7 +11198,7 @@ int ProcessPeerCerts(WOLFSSL* ssl, byte* input, word32* inOutIdx,
                             args->fatal = TLSX_CSR_InitRequest(ssl->extensions,
                                                     args->dCert, ssl->heap);
                             doLookup = 0;
-                        #if defined(WOLFSSL_TLS13) && !defined(NO_WOLFSSL_CLIENT)
```

Comment:
Same idea as below, just the opposite direction. Server built with NO_WOLFSSL_CLIENT should be able to verify a certificate status of the client, no?

Reply:
Yeah the server MAY request the client to send an OCSP response by the RFC.
Didn't think that made sense but need to support it.